### ISSUE TITLE
Create a docker image for Karma

### ISSUE THREAD

**Issue by saggu** [usc-isi-i2/Web-Karma#474]
It is helpful to have docker image for Web-Karma,  makes it easier to distribute and deploy

**Comment by saggu** [comment]
Our client Novartis want to use Karma but in docker form

**Comment by jonathanvajda** [comment]
Yeah, could this be made available on the official Docker Hub page for this group? https://hub.docker.com/u/uscisii2

I was thinking of setting up my own hub repository but I think this should belong with the official community.
